Public/Src/Utilities/Utilities/Tracing/ErrorCategoryDetails.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+
+namespace BuildXL.Utilities.Tracing
+{
+    /// <summary>
+    /// Details about errors logged for a particular error category
+    /// </summary>
+    public class ErrorCagetoryDetails
+    {
+        private int m_numErrors;
+
+        /// <summary>
+        /// The name of the first error of this category logged
+        /// </summary>
+        public string FirstErrorName { get; set; }

```

Comment:
> set;  [](start = 43, length = 6)

I think we can remove setters on these properties (everything should be set via RegisterError method, right?)

Reply:
Still need setters to modify them in that method, I made them private though

---
In reply to: [296053601](https://github.com/microsoft/BuildXL/pull/491#discussion_r296053601) [](ancestors = 296053601)